python/pyspark/mllib/util.py

```diff
@@ -169,6 +170,27 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
+        """
+        Returns a new vector with `1.0` (bias) appended to
+        the end of the input vector.
+        """
+        vec = _convert_to_vector(data)
+        if isinstance(vec, SparseVector):
+            return sp.csc_matrix(np.append(vec.toArray(), 1.0))
```

Comment:
Sorry for the slow response.  You should use scipy.sparse as in the rest of MLlib, where we (a) try to import scipy but catch the error if we do not and (b) test _have_scipy in methods and accept scipy types and only return scipy types if given scipy types.  Does that make sense?